JULABO/iocBoot/iocJULABO-IOC-01/st-common.cmd

```diff
@@ -15,13 +15,20 @@ $(IFNOTDEVSIM) $(IFNOTRECSIM) asynSetOption("L0", -1, "bits", "7")
 $(IFNOTDEVSIM) $(IFNOTRECSIM) asynSetOption("L0", -1, "parity", "even")
 $(IFNOTDEVSIM) $(IFNOTRECSIM) asynSetOption("L0", -1, "stop", "1")
 
-# Hardware flow control off
-$(IFNOTDEVSIM) $(IFNOTRECSIM) asynSetOption("L0", 0, "clocal", "Y")
-$(IFNOTDEVSIM) $(IFNOTRECSIM) asynSetOption("L0",0,"crtscts","N")
+
+## Check if hardware flow control is used
+stringiftest("HARDCNTL", "$(HARDFLOWCNTL=N)",5,"Y")
+
+# Hardware flow control
+$(IFNOTHARDCNTL) $(IFNOTDEVSIM) $(IFNOTRECSIM) asynSetOption("L0", 0, "clocal", "Y")
+$(IFNOTHARDCNTL) $(IFNOTDEVSIM) $(IFNOTRECSIM) asynSetOption("L0",0,"crtscts","N")
+
+$(IFHARDCNTL) $(IFNOTDEVSIM) $(IFNOTRECSIM) asynSetOption("L0", 0, "clocal", "N")
+$(IFHARDCNTL) $(IFNOTDEVSIM) $(IFNOTRECSIM) asynSetOption("L0",0,"crtscts","Y")
 
```

Comment:
HARDFLOWCNTR should not change "clocal", it should only change "crtscts". clocal is a different sort of hardware flow control (DCE/DTE rather than RTS/CTS) which i am not aware of being used here so we should just leave clocal as "Y" in all cases. Also we need to check on the julabo default is it looked like this may be HARDFLOWCNTL=Y